lab-jonathan/lib/greet.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(name) {
+  return 'hello ' + name;
```

Comment:
This is solid. I do want you thinking about validation though... what happens if we pass a non-string value in, or if we leave it blank?